src/importation/vcard.js

```diff
@@ -56,14 +52,23 @@ async function importData(data, options) {
     try {
       transformedContact = transform(contact)
     } catch (transformError) {
+      console.error(transformError)
       skipped.push({ contact, transformError })
       continue
     }
 
     // 2.b. Save
     try {
-      await save(transformedContact)
+      const result = await save(transformedContact)
+      if (result && result.data && result.data._rev) {
```

Comment:
I don't know how else to do it here.
Before, we did not get a result from the `save` function. But we don't know the `save` function behavior:
- is it a new created object?
- is it an updated object?
- do we get an error on `save`ing the object?

In the application code, we receive the created object as a result. To know if it is an update or a brand new object, we check the `_rev` attribute.